unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationHUD/Tests/NotificationHudTests.cs

```diff
@@ -30,7 +27,7 @@ protected override IEnumerator TearDown()
         [Test]
         public void NotificationHud_Creation()
         {
-            var views = GameObject.FindObjectsOfType<NotificationHUDView>();
+            var views = Object.FindObjectsOfType<NotificationHUDView>();
```

Comment:
Why do we swap all the `GameObject` calls for `Object` calls?

Reply:
Because those methods are really from `Object`, not `GameObject`. So using `GameObject` in the syntax works, but is misleading.